### PR TITLE
Story25 j

### PR DESCRIPTION
**Pull request by JorjaF**
__1__ Wrote Tests ____ Implemented ____ Reviewed

Neccesary checkmarks:

    [x] All Tests are Passing

    [x] The code will run locally

Type of change

    [x] New feature
    [] Bug Fix

Implements/Fixes: this add a admin merchants show page and completes story 25

    description closes # 34

Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

    [x] My code has no unused/commented out code
    [x] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

(REQUIRED)Please Include a link to a gif of your feelings about this branch

Link: https://giphy.com/gifs/jedneOuAEkcgg

### CODE REVIEW

File: app/views/admin/merchants/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<p><strong>Name:</strong> <%= @merchant.name %></p>
```
Comment (z-fitch) [inline app/views/admin/merchants/show.html.erb:1]: what is a strong div>

File: spec/features/admin/admin_merchant_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe "Admin Merchant Page", type: :feature do
+  describe "When I visit the merchant index (/admin/merchants)" do
+    # US 24
+    it "I a list of all the merchants" do
+      merchant_1 = Merchant.create!(name: "Bob's Burgers")
+      visit admin_merchants_path
```
Comment (z-fitch) [inline spec/features/admin/admin_merchant_spec.rb:8]: per last PR maybe add more merchants and test for them for more rounded tests

**Review by z-fitch — APPROVED** [review]

**Review by Daniel-Gallagher92 — APPROVED** [review]
Beef up these tests and we are good!!

File: app/views/admin/merchants/show.html.erb (hunk 1 above)
Comment (ethanrossblack) [reply app/views/admin/merchants/show.html.erb:1]: `<strong>` bolds HTML text. `<b>` works, too.

**Comment by JorjaF** [comment]
i did a new PR

**Pull request closed without merging**